web-console/package.json

```diff
@@ -170,5 +171,8 @@
     "webpack-bundle-analyzer": "^4.4.1",
     "webpack-cli": "^4.6.0",
     "webpack-dev-server": "^3.11.2"
+  },
+  "overrides": {
+    "@types/react": "$@types/react"
```

Comment:
what does this do and why is it needed?

Reply:
Some dependencies include `@types/react` as a dependency but pinned to specific versions. This forces those transitive dependency versions to resolve to the version specified in our package file.